src/components/organisms/Containers/CalendarContainer/CalendarContainer.tsx

```diff
@@ -1,11 +1,31 @@
-import TaskFilters from "../../TaskFilters/TaskFilters";
+import { useLayoutEffect, useMemo } from "react";
+import { useAppDispatch, useAppSelector } from "store/hooks";
+
+import { fetchTaskArray, selectIsLoading } from "features/todo/todoSlice";
+
+import TaskFilterBar from "../../TaskFilters/TaskFilterBar";
 import CalendarTaskList from "../../Lists/CalendarTaskList/CalendarTaskList";
 
+import { getTodayString } from "utils/helpers";
+
 const CalendarContainer = () => {
+  const dispatch = useAppDispatch();
+  const isLoading = useAppSelector(selectIsLoading);
+
+  const filters = useMemo(() => {
+    return {
+      date: `ge${getTodayString()}`,
+    };
+  }, []);
+
+  useLayoutEffect(() => {
+    dispatch(fetchTaskArray(filters));
+  }, [dispatch, filters]);
+
   return (
     <main className="w-full mr-[0px]">
-      <TaskFilters />
-      <CalendarTaskList />
+      <TaskFilterBar filters={filters} />
+      {!isLoading && <CalendarTaskList />}
```

Comment:
maybe some loader when is loading?

Reply:
loader is in TaskCreator component, but CalendarContainer and FilteredContainer don't include it, only TaskContainer does. Should I change it? It usually loads fast so I don't know if it needs one

Reply:
myślę że byłoby dobrze mieć, chociaż nie wiem jak to wygląda - jeli nie będzie za dużo tych loaderów to bym dodał